src/sidebar/img/find.svg

```diff
@@ -1,22 +1,6 @@
-<?xml version="1.0"?>
```

Comment:
This new icon doesn't align with the other sidebar ones (bookmarks, history, synced tabs). Could we restore the old one? However, I like the removal of #standard/#inverted :)